src/gradio-station-compare.vala

```diff
@@ -81,11 +81,11 @@ namespace Gradio{
 					break;
 				}
 				case Compare.STATE: {
-					result = (strcmp(stationA.state, stationB.state) * -1);
+					result = strcmp(stationA.state, stationB.state);
 					break;
 				}
 				case Compare.DATE: {
-					result = (strcmp(stationA.clicktimestamp, stationB.clicktimestamp));
+					result = strcmp(stationA.clicktimestamp, stationB.clicktimestamp);
```

Comment:
Why this? You are going to swap the ascending and descending sort order. 

You compare two different stations.
1 -> Station A > Station B
0 -> Station A = Station B
-1 -> Station A < Station B

Reply:
Yes, but I make it consistent with the integer compares, e.g. the votes count. Now votes sort the other way around from the string-based compares, because the multiplication by -1 is there for the string based compares. That was my reason for removing the * -1, so everything works as a user would expect.